### ISSUE TITLE
Data Quality Fail Causes Collision Pipeline Fail

### ISSUE THREAD

**Issue by scann0n** [CityofToronto/bdit_data-sources#526]
The "from MOVE to postgres" portion of the collisions pipeline failed because it couldn't refresh the `collisions_replicator.events` mat view.

`collisions_replicator.events` contains one record for every collision. There's a unique index that is created using the `collision_no` field to ensure this. The pipeline failed because the unique index had duplicates in it!

I found the source of the duplicate (without looking hard because the answer was right in the airflow log):
![airflow_col_pipeline](https://user-images.githubusercontent.com/94873142/218882344-9fc46b22-3cfd-4197-92db-1b79a0a23b20.png)

Anywho according to the `collisions_replicator."ACC"` table there were two records with the same `"ACCNB"` that had inconsistent values for collision details (like `"ACCTIME"` and street names). So even when you used `SELECT DISTINCT` you would end up with two records for the same `"ACCNB"` because of these inconsistencies.

The inconsistencies were fixed by members of the MOVE team and @scann0n manually re-ran the functions and that worked, and as expected the pipeline is up and running again.

**TL; DR:** 
If the pipeline fails because you're violating a unique constraint take a look to see if there are multiple records with inconsistent info relating to the collision event (like the time it occurred or street name spelling).
